src/main/java/net/cyberkitsune/prefixchat/ChatListener.java

```diff
@@ -101,6 +103,14 @@ public void playerChat(AsyncPlayerChatEvent evt) {
 			target_channel = default_channel;
 		}
 
+		// If message is longer than 200 characters (max is 256) reminder player they can append
+		if (evt.getMessage().length()>=200) {
+			if (!dashReminded.contains(evt.getPlayer().getDisplayName())) {
+				evt.getPlayer().sendMessage(LocalizedString.get("dashreminder", evt.getPlayer().getLocale()));
+				dashReminded.add(evt.getPlayer().getDisplayName());
+			}
+		}
+
```

Comment:
This will fire if a user can't send a message, and before the message instead of after too. Move this block to right after line 129.

Reply:
It didn't matter if the block was at the top of everything, or below at the very bottom after the line where the line is sent, the reminder was still sent before the message was shown in the chat for the player.
![2IrEMTR](https://user-images.githubusercontent.com/65049299/82596150-b4733600-9b5b-11ea-8f81-e344aacae75f.png)